.github/workflows/deploy.yml

```diff
@@ -198,6 +201,7 @@ jobs:
       - uses: actions/upload-artifact@v3
         with:
           path: ./wheelhouse/*.whl
+          name: aer-deploy-gpu-build-cuda12-${{ matrix.os }}
       - name: Publish package distributions to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Deployed here.  Also, it's `upload-artifact@v3.`

Reply:
Bumped to 4 and added "separate" in d9d7fb78